text/0060-tasks-api.md

```diff
@@ -126,11 +140,15 @@ As writing is asynchronous for most of Meilisearch's operations, this API makes
 
 | name    | description  |
 | -----   | ------------ |
-| dumpUid | The generated uid of the dump |
+| dumpUid | The generated uid of the dump. `null` when the task status is enqueued or processing. |
```

Comment:
```suggestion
| dumpUid | The generated uid of the dump. `null` when the task status is enqueued, processing, canceled or, failed. |
```

Reply:
Signal to @maryamsulemani97. We are adding precision here that you may want to mention in the related API reference.